rs/sns/governance/src/types.rs

```diff
@@ -2588,6 +2588,7 @@ impl From<MintSnsTokens> for Action {
     }
 }
 
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub enum Wasm {
```

Comment:
Dumb question, why can't we just use this type in the API?  It seems like a nice type, and it covers both cases

Reply:
We've introduced this type to support backwards compatibility for upgrade_sns_controlled_canister proposals, which used to only allow for embedded Wasms. 

Now we have a better pattern, that is more general and avoids redundantly sending Wasms around subnets, so I would like to enforce that the clients use the better pattern.